### PR TITLE
More CSP whitelist rules

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Some more CSP rules after log watching

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1922** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1922" target="_blank">Graphite</a> 
    * **PR #1941** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1941" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1941?utm_source=stack-comment).

**Review by gustaveen — APPROVED** [review]
LGTM!

**Comment by alebedev** [comment]
@alebedev queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1941).

**Comment by alebedev** [comment]
@alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1941).

File: apps/store/next-csp.config.js [hunk 1]
```diff
@@ -126,6 +129,7 @@ const frameSrc = [
 
 const ContentSecurityPolicy = `
   default-src 'self';
+  child-src 'blob:' 'self',
```
Comment (gustaveen) [inline apps/store/next-csp.config.js:132]: Should end with semicolon right? @alebedev

**Pull request merged**